crates/workspace/src/workspace_settings.rs

```diff
@@ -53,20 +53,20 @@ pub struct ActivePanelModifiers {
     /// When set to 1.0, the active pane has the same size as others,
     /// but when set to a larger value, the active pane takes up more space.
     ///
-    /// Default: `1.0`
+    /// Default: 1.0
```

Comment:
There seems to be a bunch of other places where code values are tick-escaped, changing them all creates quite a diff, so not sure we should touch this at all.

Let's omit the changes in this file?

Reply:
If so, let's remove it.